assets/components/LogViewer/LogActionsToolbar.vue

```diff
@@ -65,17 +65,56 @@
           {{ $t("toolbar.show", { std: "STDERR" }) }}
         </a>
       </li>
+      <li class="line"></li>
```

Comment:
I am confused here. Maybe I missed it last time. Doesn't this also need to be hidden?

Reply:
I have changed it to `<template>` tag! so everything gets inside a single control flow!